version.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh
+
+set -eu
```

Comment:
I wonder if this makes sense, I think if git is not available in a build system it'll make the script fail. Maybe I should remove this altogether or reverse the conditional to check for the non git case first.